File: openapi-generator/scalaz/src/main/scala/org/openapitools/client/api/InlineObject17.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import InlineObject17._

case class InlineObject17 (
  /* End of the time range, specified in milliseconds UNIX time. */
  endMs: Integer,
/* Group ID to query. */
  groupId: Long,
/* Beginning of the time range, specified in milliseconds UNIX time. */
  startMs: Integer)

object InlineObject17 {
  import DateTimeCodecs._

  implicit val InlineObject17CodecJson: CodecJson[InlineObject17] = CodecJson.derive[InlineObject17]
  implicit val InlineObject17Decoder: EntityDecoder[InlineObject17] = jsonOf[InlineObject17]
  implicit val InlineObject17Encoder: EntityEncoder[InlineObject17] = jsonEncoderOf[InlineObject17]
}

File: openapi-generator/dart-jaguar/lib/model/inline_object17.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'inline_object17.jser.dart';

class InlineObject17 {
   /* End of the time range, specified in milliseconds UNIX time. */
  @Alias('endMs')
  final int endMs;
   /* Group ID to query. */
  @Alias('groupId')
  final int groupId;
   /* Beginning of the time range, specified in milliseconds UNIX time. */
  @Alias('startMs')
  final int startMs;
  

  InlineObject17(
      

{
    
     this.endMs = null,  
     this.groupId = null,  
     this.startMs = null 
    }
  );

  @override
  String toString() {
    return 'InlineObject17[endMs=$endMs, groupId=$groupId, startMs=$startMs, ]';
  }
}

@GenSerializer(nullableFields: true)
class InlineObject17Serializer extends Serializer<InlineObject17> with _$InlineObject17Serializer {

}
